README.md

```diff
@@ -199,9 +199,9 @@ The remaining tests in wave 6 imply:
     - `my_priority`, which represents a category that the `Vendor` wants to receive
     - `their_priority`, which represents a category that `other_vendor` wants to receive
   - The best item in my inventory that matches `their_priority` category is swapped with the best item in `other_vendor`'s inventory that matches `my_priority`
+    - If `other_vendor` has no item that matches `my_priority` category, swapping does not happen, and it returns `False`
```

Comment:
Was this change accidentally committed?

If this change wasn't needed then take care to remove it from the list of files being added and committed. You can unstage a file for commit if you haven't run `git commit` yet and you can look at how to locally revert a commit if you already added/committed the file. 

In industry, when you write code and open a PR it means you're asking to merge code into the main branch. As a result, you will not be permitted to merge in changes that are not part of your feature work because it will cause noise in the commit history or bring in unnecessary changes to the codebase. 

It's best practice to run git status to see what changes will be committed before you commit so you can catch stray changes like this.